sycl/source/detail/platform_impl.cpp

```diff
@@ -100,11 +100,11 @@ vector_class<platform> platform_impl::get_platforms() {
   for (unsigned int i = 0; i < Plugins.size(); i++) {
```

Comment:
I wonder why we don't use range-based for here...

Reply:
I guess it is C++98 habits